torchok/constructor/registry.py

```diff
@@ -16,15 +26,18 @@ def __init__(self, name: str):
             name: Component name.
         """
         self.__name = name
-        self.__class_dict = dict()
+
+        self.__entrypoints = {}  # mapping of class/function names to entrypoint fns
+        self.__module_to_objects = defaultdict(set)  # dict of sets to check membership of class/function in module
+        self.__object_to_module = {}  # mapping of class/function names to module names
```

Comment:
Where we use self.__object_to_module?

Reply:
In comment `module names` why it can be modules not only one module?

Reply:
Now it is not used but let it be for the future. In comment there is an mistake. I'll fix it